index.js

```diff
@@ -49,6 +50,11 @@ module.exports = function percySnapshot(b, name, options) {
   });
 };
 
+// To mock the test case
+module.exports.request = async function request(data) {
```

Comment:
we can spyOn(utils, postSnapshot) as well

Reply:
This will not work as utils does not have setter and getter it will need the cli update and code change for the same.